ts/src/ace.ts

```diff
@@ -293,7 +293,7 @@ export default class ace extends Exchange {
          * @see https://github.com/ace-exchange/ace-official-api-docs/blob/master/api_v2.md#oapi-api---trade-data
          * @param {string} symbol unified symbol of the market to fetch the ticker for
          * @param {object} [params] extra parameters specific to the ace api endpoint
-         * @returns {object} a [ticker structure]{@link https://github.com/ccxt/ccxt/wiki/Manual#ticker-structure}
+         * @returns {object} a [ticker structure]{https://docs.ccxt.com/#/?id=ticker-structure}
```

Comment:
@pcriadoperez I think you accidentally removed the @link tag

Reply:
good catch!